### PR TITLE
Feature/team page about section

### PR DESCRIPTION
**Pull request by sbounadja-at-speer**

### CODE REVIEW

File: src/view/pages/Team/sections/About.tsx [hunk 1]
```diff
@@ -1,79 +1,182 @@
...
+`;
+
+const AboutUsTextWrapper = styled.div`
+     width: 365px;
+    @media ${MediaQuery.tablet} {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        width: 675px;
+    }
```
Comment (konner-at-speer) [inline src/view/pages/Team/sections/About.tsx:180]: For tablet screens and larger, let's add `padding-left: 25px` to add additional space.

File: src/view/pages/Team/sections/About.tsx [hunk 2]
```diff
@@ -1,79 +1,182 @@
...
     @media ${MediaQuery.tablet} {
-        width: 55%;
-        margin-bottom: 38px;
+        width: 90%;
+        
     }
 
 `;
 
 const Content = styled.p`
```
Comment (konner-at-speer) [inline src/view/pages/Team/sections/About.tsx:129]: Double-check font-size and margins here

**Review by konner-at-speer — CHANGES_REQUESTED** [review]
Overall, looks good! Make sure you double-check the margins and font sizes for all the elements. Also, I noticed the 'Speer' image ( **S** specifically) seems cut off a bit. Can you investigate this? May have to take this up with design.
Screen Shot 2022-03-21 at 2 54 38 PM

File: src/view/pages/Team/sections/About.tsx [hunk 3]
```diff
@@ -1,79 +1,182 @@
...
+            <AboutUsImageWrapper>
+                <Image src={IMAGES.teamAboutUsGroup} alt="About us group image" />
+                <SpeeImage src={IMAGES.teamSpeer} alt="Speer team speer word image" />
+            </AboutUsImageWrapper>
         </Container>
     );
 };
 
-const Container = styled.div`
+const Button = styled.a`
```
[the remark this answers is not in the document]
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx]: has been replaced with button element.

File: src/view/pages/Team/sections/About.tsx [hunk 4]
```diff
@@ -1,79 +1,182 @@
...
-    margin: 0px auto 100px auto;
 
-    @media ${MediaQuery.mobileL} {
-        width: 370px;
+    width: 11.5em;
+    height: 2.85em;
+    margin-top: 10px;
+
+    border: 2px solid ${Palette.BLACK};
+    background-color: ${Palette.BLACK};
```
[the remark this answers is not in the document]
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx]: has been replaced with {Palette.DARK_GRAY}

File: src/view/pages/Team/sections/About.tsx [hunk 5]
```diff
@@ -1,79 +1,182 @@
...
+
+const Image = styled.img<{ src: string }>`
+    max-width: 100%;
+    height: 292px;
+    position: relative;
+    right: 15px;
+    top: 60px;
+    @media ${MediaQuery.tablet} {
+    width:100%;
+    height:100%;
```
[the remark this answers is not in the document]
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx:73]: formatting fixed.

File: src/view/pages/Team/sections/About.tsx [hunk 6]
```diff
@@ -1,79 +1,182 @@
...
+        top: 0;
     }
+`;
+
+const SpeeImage = styled.img<{ src: string }>`
```
[the remark this answers is not in the document]
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx]: naming fixed.

File: src/view/pages/Team/sections/About.tsx [hunk 7]
```diff
@@ -1,79 +1,182 @@
...
+    text-align: left;
     font-family: 'Poppins-Medium';
     font-size: 35px;
     color: ${Palette.DARK_GRAY};
     margin-bottom: 19px;
     @media ${MediaQuery.tablet} {
         font-family: 'Poppins-Regular';
         font-size: 65px;
-        margin-bottom: 27px;
+        margin-bottom: 25px;
```
[the remark this answers is not in the document]
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx]: values has been replaced with proper onces.

File: src/view/pages/Team/sections/About.tsx (hunk 2 above)
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx:136]: margin fixed

File: src/view/pages/Team/sections/About.tsx (hunk 1 above)
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx]: added some margin there.

File: src/view/pages/Team/sections/About.tsx [hunk 8]
```diff
@@ -1,79 +1,195 @@
...
+    width: 96%;
+    box-sizing: border-box;
+
     font-size: 35px;
     color: ${Palette.DARK_GRAY};
-    margin-bottom: 19px;
+    margin-bottom: 20px;
     @media ${MediaQuery.tablet} {
-        font-family: 'Poppins-Regular';
+        font-family: 'Poppins-Medium';
```
Comment (konner-at-speer) [inline src/view/pages/Team/sections/About.tsx:112]: The font here is too bold. I'd suggest you revert this change back to `font-family: 'Poppins-Regular';` for all screen sizes.

**Review by konner-at-speer — CHANGES_REQUESTED** [review]
Just a few additional changes. Also, do we need those **about_us** PNG/WEBP assets anymore if we're using svg?

File: src/view/pages/Team/sections/About.tsx (hunk 8 above)
Comment (sbounadja-at-speer) [reply src/view/pages/Team/sections/About.tsx]: done.

**Comment by sbounadja-at-speer** [comment]
> Just a few additional changes. Also, do we need those **about_us** PNG/WEBP assets anymore if we're using SVG's?

we have to remove the related object from `imagesWithAdditionalFormats` to be able to delete those files.

**Comment by sbounadja-at-speer** [comment]
> > Just a few additional changes. Also, do we need those **about_us** PNG/WEBP assets anymore if we're using SVG's?
> > > we have to remove the related object from `imagesWithAdditionalFormats` to be able to delete those files.
> 
> Can we remove the related object from `imagesWithAdditionalFormats`?

assets has been removed.

**Pull request merged**